include/myjansson.h

```diff
@@ -7,7 +7,7 @@
 #  include <vector> 
 #  include <cstdlib>
 #  ifndef LEGACY_JANSSON
```

Comment:
I think jansson has been removed from all our newer platforms, so these checks can be removed.

Reply:
The LEGACY_JANSSON checks are too pervasive; rather than removing now, shall we record an issue instead, in the interest of a sooner release?

Reply:
Sounds good. Created https://github.com/irods/irods_netcdf/issues/35 for this.